docs/next/guides/cell-features/formatting-cells.md

```diff
@@ -94,8 +94,21 @@ To enable the custom borders feature, set the `customBorders` option. This can e
 
 To initialize Handsontable with predefined custom borders, provide the cell coordinates and border styles in form of an array:
 
-- with row/col pairs: `{ row: 2, col: 2, left: { /*...*/ } }`
-- or with range details: `{ range: { from: { row: 1, col: 1 }, to: { row: 3, col: 4 } }, left: { /*...*/ } }`
+- with row/col pairs: `{ row: 2, col: 2, start: { /*...*/ } }`
+- or with range details: `{ range: { from: { row: 1, col: 1 }, to: { row: 3, col: 4 } }, start: { /*...*/ } }`
+
+Depending on the form of the cell coordinates you picked, you can use the following options to change the border styles:
+
+::: warning
+The `left` and `right` object keys are renamed to `start` and `end`. The change allows defining border styles using the same API for both [layout direction](@/guides/internationalization/layout-direction.md) modes - left to right or right to left. The keys `left` and `right` will still work for backward compatibility. However, they can work only in LTR and cannot be mixed with keys `start` and `end`.
+:::
+
+| Option | Description |
+| --- | --- |
+| `start: { color: String, width: Number }` | Defines the border style of the _left_ border or the _right_ border when the table is configured to work with [RTL layout direction](@/guides/internationalization/layout-direction.md) mode. |
+| `end: { color: String, width: Number `    | Defines the border style of the _right_ border or the _left_ border when the table is configured to work with [RTL layout direction](@/guides/internationalization/layout-direction.md) mode. |
+| `top: { color: String, width: Number `    | Defines the border style of the top border.    |
+| `bottom: { color: String, width: Number ` | Defines the border style of the bottom border. |
```

Comment:
I propose to add just a link here and expand the information in JSDoc:

```suggestion
For the list of available settings and methods, visit the [API reference](@/api/custom-borders.md#options).
```

Reply:
I fixed it within f923d9ec45384ae50e2992c7fa96eab6553f212a.